apps/web/src/modules/dao/components/About/mdRender.css.ts

```diff
@@ -0,0 +1,120 @@
+import { globalStyle, style } from '@vanilla-extract/css'
+
+export const daoDescription = style({
+  wordBreak: 'break-word',
+})
+
+const mqMobile = '@media (max-width: 768px)'
+
+globalStyle(`${daoDescription} :is(h1)`, {
+  fontSize: '2.6rem',
+  fontWeight: 700,
+  borderBottom: '1px solid #f0f0f0',
+  lineHeight: '1.8',
+  marginBottom: 18,
+  '@media': {
+    '(max-width: 768px)': {
+      fontSize: '2rem',
+    },
+  },
+})
+
+globalStyle(`${daoDescription} :is(h2)`, {
+  fontSize: '2.1rem',
+  fontWeight: 700,
```

Comment:
not fontWeight and marginBottom can be atoms

Reply:
Read above